### ISSUE TITLE
datasets

### ISSUE THREAD

**Issue by songbohui** [arealgoodname/DiffCap#5]
How can I get the paths to the training set and validation set in the code, and where can I download the relevant files?
